server/liveblog/posts/utils.py

```diff
@@ -151,3 +151,42 @@ def get_main_item(post):
         )
 
     return main_item
+
+
+def check_content_diff(updates, original):
```

Comment:
we should cover this one with tests please.

Reply:
I've added tests for this in `server/liveblog/posts/tests/posts_test.py`